.github/workflows/publish.yml

```diff
@@ -21,3 +21,28 @@ jobs:
       uses: JS-DevTools/npm-publish@v1
       with:
         token: ${{ secrets.NPM_TOKEN }}
+  github:
+    name: GitHub release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Get release version
+      id: get_version
+      run: |
+        export PACKAGE_VERSION=$(cat package.json | grep 'version' | sed 's/[ \",:]//g' | sed 's/version//')
+        echo "::set-output name=version::$PACKAGE_VERSION"
+    - name: Create and push git tag
+      uses: actions-ecosystem/action-push-tag@v1
+      with:
+        tag: ${{ steps.get_version.outputs.version }}
+        message: ${{ steps.commit.outputs.git-message }}
+    - name: Create release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ steps.get_version.outputs.version }}
+        release_name: Release ${{ steps.get_version.outputs.version }}
+        draft: false
+        prerelease: false
```

Comment:
These are really needed? Explicitly could be better, but I'm not sure in this context, seems that It's very clear that by releases will not be prereleased and drafted by default :thinking:

Reply:
Actually I would be in favor of this (assuming these values are the defaults), the only reason they're there in the main repo is because I copied them from a project where I currently have `prerelease: true`. We can also update it in the main repo for consistency.

Reply:
OK, I'm opening a pull request in the main repo to change this and underscores in the step id.

Reply:
I removed these now as [the Action's documentation](https://github.com/actions/create-release/tree/c38d3a140cc22e67e265c5d5b6b4888d1f02533f#inputs) specifies that both are `false`  by default.